lib/stat_tracker.rb

```diff
@@ -113,4 +113,47 @@ def worst_coach(season)
 
     coach_results.min_by { |k, v| v }[0]
   end
+
+  def calculate_team_accuracies(season)
+    team_accuracies = {}
+
+    game_teams_in_season = @game_teams_data.select do |game_team|
+      game_id = game_team[:game_id]
+      game = @games_data.find { |game| game[:game_id] == game_id }
+
+      game && game[:season] == season
+    end
+
+    game_teams_in_season.each do |game_team|
+      team_id = game_team[:team_id]
+      goals = game_team[:goals].to_i
+      shots = game_team[:shots].to_i
+
+      if shots > 0
+        accuracy_ratio = goals.to_f / shots
+        team_accuracies[team_id] ||= []
```

Comment:
you can drop line 134 if you set up your `team_accuracies` hash to have a default value of `[ ]`

Reply:
done!